src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileParser.cs

```diff
@@ -73,12 +73,15 @@ private XamlFileDefinition ParseFile(string file)
 
 		private XmlReader ApplyIgnorables(string file)
 		{
+			var adjusted = File.ReadAllText(file, Encoding.UTF8);
+			var containsXBind = adjusted.Contains("{x:Bind");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'adjusted.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5467800)